awsfindingsmanagerlib/awsfindingsmanagerlib.py

```diff
@@ -357,10 +362,15 @@ def is_matching_rule(self, rule: Rule) -> bool:
             self.match_if_set(self.security_control_id,
                               rule.security_control_id),
             self.match_if_set(self.control_id, rule.rule_or_control_id),
-            self.match_if_set(self.rule_id, rule.rule_or_control_id)
+            self.match_if_set(self.rule_id, rule.rule_or_control_id),
```

Comment:
please update the function documentation above

Reply:
Done :)